manifests/install.pp

```diff
@@ -0,0 +1,9 @@
+# @api private
+class htcondor::install {
+  assert_private()
+
+  package { 'condor':
```

Comment:
It is preferable to use `ensure_packages` so as to avoid a possible resource conflict.

Reply:
changed to ensure_packages to avoid issues with more than 1 definition on the resource